docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,259 @@
+
+### EO data download services 
+
+[**European Space Agency's SciHub**](https://scihub.copernicus.eu/dhus/#/home) provides worldwide Sentinel 2 L1C and L2A and Sentinel 1 SLC, GRD , RAW and OCN products. It requires free [SciHub registration](https://scihub.copernicus.eu/dhus/#/self-registration) and many of the data is in "Long term archive" and cannot be downloaded directly, but needs to be requested. Download is limited to 2 concurrent processes per user.
+
+[**FinHub**](https://finhub.nsdc.fmi.fi/#/home) is the Finnish national mirror of SciHub; others national mirrors also exist. It covers Finland and the Baltics and offers Sentinel 2 L1C (but not L2A) and Sentinel 1 SLC, GRD and OCN products and requires own registration. Finhub does not have concurrent download limitations.
+
+!!! default  ""  
+    **Both of the above** provide a similar Graphical User Interface (GUI) and Application Programming Interface (API) to access the data. You can also use for example the [sentinelsat](https://sentinelsat.readthedocs.io/en/stable/) tool for downloading data from open access hubs. See also [CSC examples for SciHub and FinHub data download](https://github.com/csc-training/geocomputing/blob/master/python/sentinel/sentinelsat_download_from_finhub_and_scihub.py).
+
+
+[**USGS EarthExplorer**](https://earthexplorer.usgs.gov/) provides lots of different US related datasets, also worldwide Landsat mission datasets. It requires free [EarthExplorer registration](https://ers.cr.usgs.gov/register). Data can be browsed and downloaded via GUI in web interface and bulk download.
+    
+[**NASA Earthdata**](https://search.earthdata.nasa.gov) provides among many others harmonized Landsat 8 and Sentinel-2 dataset. It requires [NASA EarthData registration](https://urs.earthdata.nasa.gov/users/new) and download is possible via Graphical web interface and bulk download.
+
+**Finnish Sentinel image mosaics** are provided by the [Finnish Environment Institute, SYKE](https://www.syke.fi/en-US) and the [Finnish Meteorological Institute, FMI](https://en.ilmatieteenlaitos.fi/) and partly available in Puhti: `/appl/data/geo/sentinel`. Among others, [Sentinel-2 image index mosaics](https://ckan.ymparisto.fi/dataset/sentinel-2-image-index-mosaics-s2ind-sentinel-2-kuvamosaiikit-s2ind)  and [Sentinel-1 SAR-image mosaics](https://ckan.ymparisto.fi/dataset/sentinel-1-sar-image-mosaic-s1sar-sentinel-1-sar-kuvamosaiikki-s1sar) are available.
+
+[**Google Cloud Storage**](https://cloud.google.com) provides <a href="https://cloud.google.com/storage/docs/public-datasets/sentinel-2" aria-label="Sentinel-2 data on Google cloud storage">Sentinel 2: L1C</a> 
```

Comment:
```suggestion
[**Google Cloud**](https://cloud.google.com) provides some [public EO datasets in Google Cloud Storage](https://cloud.google.com/storage/docs/public-datasets), inlcuding Sentinel-2 L1C and Landsat Collection 1 data. Google Cloud also offers the possibility for virtual machines, with both CPU and GPU.
```

Reply:
I think that Google should also be in same category as AWS, one can get VMs also there.